lib/fastlane/plugin/test_center/actions/multi_scan.rb

```diff
@@ -170,9 +170,10 @@ def self.build_for_testing(scan_options)
       end
 
       def self.prepare_scan_options_for_build_for_testing(scan_options)
+        build_options = scan_options.merge(build_for_testing: true).reject { |k| k == :test_without_building }
```

Comment:
won't this always override what ever value we passed to `test_without_building`? we use multi_scan in two scenarios - one is when it builds and tests together and another when we build first and then run specific subset of test and we need to pass `test_without_building: true`

Reply:
No, `multi_scan` only builds if it finds that `:test_without_building` is true or if `:skip_build` is true. See [here](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/452c251902499ade367a2cf415cafe16c9edb80d/lib/fastlane/plugin/test_center/actions/multi_scan.rb#L118).